saas/axops/src/applatix.io/axops/configuration/configuration.go

```diff
@@ -23,16 +27,42 @@ type ConfigurationContext struct {
 	Key  string
 }
 
+type SecretResult struct {
+	SecretData     map[string]string `json:"data"`
+	SecretMetadata map[string]string `json:"metadata"`
+}
+
 const (
 	ConfigurationStrRegex = "^%%config\\.([^ ]*)\\.([-0-9A-Za-z_]+)\\.([-0-9A-Za-z_]+)%%$"
 )
 
+var MaxRetryDuration time.Duration = 60 * time.Second
+
+var retryConfig *restcl.RetryConfig = &restcl.RetryConfig{
+	Timeout: MaxRetryDuration,
+}
+
 func GetConfigurations(params map[string]interface{}) ([]ConfigurationData, *axerror.AXError) {
 	configs := []ConfigurationData{}
 	axErr := utils.Dbcl.Get(axdb.AXDBAppAXOPS, ConfigurationTableName, params, &configs)
 	if axErr != nil {
 		return nil, axErr
 	}
+	for idx, config := range configs {
```

Comment:
I thought the plan was to never return the kubernetes secret from the api so that anonymous users cannot access it.